crates/cli/src/utils/cmd.rs

```diff
@@ -173,7 +173,12 @@ pub fn has_different_gas_calc(chain_id: u64) -> bool {
                 NamedChain::Moonbase |
                 NamedChain::Moonbeam |
                 NamedChain::MoonbeamDev |
-                NamedChain::Moonriver
+                NamedChain::Moonriver |
+                NamedChain::Mandala |
+                NamedChain::KaruraTestnet |
+                NamedChain::AcalaTestnet |
+                NamedChain::Karura |
+                NamedChain::Acala
```

Comment:
Please make sure to alphabetically sort the list, this makes it easier to maintain

Reply:
See my note on `NamedChain::Mandala` in `alloy-chains`